.cbmc-batch/jobs/Makefile.common

```diff
@@ -70,6 +70,11 @@ CBMCFLAGS +=  --object-bits $(CBMC_OBJECT_BITS)
 DEFINES += -DCBMC_OBJECT_BITS=$(CBMC_OBJECT_BITS)
 DEFINES += -DCBMC=1
 
+################################################################
+# Don't execute deep checks by default
+AWS_DEEP_CHECKS ?= 0
+DEFINES += -DAWS_DEEP_CHECKS=$(AWS_DEEP_CHECKS)
```

Comment:
There are two ways to go here: let it be 0, which means that you could use it in a ```if```, or leave it undefined, whihc lets you use it in ```ifdef``` as well.  Either is fine, but we should be consistent with what the rest of the project does for flags like this.